docs/pages/api-reference/liveblocks-react-ui.mdx

```diff
@@ -1325,8 +1327,9 @@ declare global {
 
 Your activities data is now correctly typed in inline functions.
 
-```tsx highlight="4-5"
+```tsx highlight="5-6"
```

Comment:
You can use the new `// +++` highlight markers here, instead of adding these line numbers manually.